packages/breadboard-cli/src/commands/import.ts

```diff
@@ -82,7 +82,7 @@ export const importGraph = async (url: string, options: ImportOptions) => {
 
       const board = apiRef;
```

Comment:
maybe `const { board } = apiRef` and it will get rid of one of the board.board.board's

Reply:
```suggestion
      const { board } = apiRef;
```